source_handlers_test.go

```diff
@@ -1305,6 +1305,57 @@ func TestSourceEditBadRequest(t *testing.T) {
 	templates.BadRequestTest(t, rec)
 }
 
+func TestSourceDeleteWithOwnershipWhenUserIsNotAllowedToDelete(t *testing.T) {
+	testutils.SkipIfNotRunningIntegrationTests(t)
+	testutils.SkipIfNotSecretStoreDatabase(t)
+
+	accountNumber := "112567"
+	userIDWithOwnRecords := "user_based_user"
+	otherUserIDWithOwnRecords := "other_user"
```

Comment:
for this user you dont create own records (I know that it is not needed), but then this user is not "WithOwnRecords", or?)